_includes/v22.2/start-in-docker/mac-linux-steps-single-node.md

```diff
@@ -0,0 +1,128 @@
+    - Accepts connections to the DB Console on IP address 172.18.0.3 (which resolves to the hostname `roach-single` within the Docker container) on port 8080.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    docker run -d \
+              --env COCKROACH_DATABASE={DATABASE_NAME} \
+              --env COCKROACH_USER={USER_NAME} \
+              --env COCKROACH_PASSWORD={PASSWORD} \
+              --name=roach-single \
+              --hostname=roach-single \
+              --ip=172.18.0.3 \
+              --advertise-addr=172.18.0.3 \
+              -p 26257:26257 -p 8080:8080 \
+              -v "roach-single:/cockroach/cockroach-data" \
+              -v "~/init-scripts:/docker-entrypoint-initdb.d" \
+              cockroachdb/cockroach:latest start-single-node
+    ~~~
+
+    By default, a `certs` directory is created and CockroachDB starts in secure mode. To prevent this, add the `--insecure` flag after the `start-single-node` sub-command.
+
+    {{site.data.alerts.callout_info}}
+    The `COCKROACH_DATABASE`, `COCKROACH_USER`, and `COCKROACH_PASSWORD` environment variables and the contents of the `/docker-entrypoint-initdb.d` directory are ignored if you use `cockroach start` rather than `cockroach start-single-node`. They are also ignored if the `/cockroach/cockroach-data` directory within the container is not empty.
+    {{site.data.alerts.end}}
+
+1. Perform a one-time initialization of the cluster:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    docker exec -it roach-single ./cockroach init --insecure
+    ~~~
+
+    You'll see the following message:
+
+    ~~~
+    Cluster successfully initialized
+    ~~~
+
+    At this point, the cluster node also prints helpful [startup details](cockroach-start.html#standard-output) to its log. For example, the following command retrieves node 1's startup details:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    docker exec -it roach-single grep 'node starting' cockroach-data/logs/cockroach.log -A 11
+    ~~~
+
+    The output will look something like this:
+
+    ~~~
+    CockroachDB node starting at {{ now | date: "%Y-%m-%d %H:%M:%S.%6 +0000 UTC" }}
+    build:               CCL {{page.release_info.version}} @ {{page.release_info.build_time}} (go1.12.6)
+    webui:               http://roach-single:8080
+    sql:                 postgresql://root@roach-single:26257?sslmode=disable
+    client flags:        /cockroach/cockroach <client cmd> --host=roach-single:26257
+    logs:                /cockroach/cockroach-data/logs
+    temp dir:            /cockroach/cockroach-data/cockroach-temp273641911
+    external I/O path:   /cockroach/cockroach-data/extern
+    store[0]:            path=/cockroach/cockroach-data
+    status:              initialized new cluster
+    clusterID:           1a705c26-e337-4b09-95a6-6e5a819f9eec
+    nodeID:              1
+    ~~~
+
```

Comment:
I think we can add another step here about accessing the sql shell of the single-node server.

```
docker exec -it roach-single ./cockroach sql --url="<sql-address>"
```

where the `sql-address` is the `sql` in the above output (e.g. `postgresql://root@127.0.0.1:26257/defaultdb?sslcert=certs%2Fclient.root.crt&sslkey=certs%2Fclient.root.key&sslmode=verify-full&sslrootcert=certs%2Fca.crt`)

Reply:
Done. I actually added another "step" to connect to the cluster.